TicTacToeThree.js

```diff
@@ -0,0 +1,71 @@
+const squareDivs = document.querySelectorAll('.squares');
+
+
+//Variables
+let xIsNext = true;
+let playerTurn;
+let i;
+//let gameEnd = setInterval(gameTimer, 100000)
+
+//let squareOne = squareOne[0].classList.length;
+//let squareTwo = squareOne[1].classList.length;
+//let squareThree = squareOne[2].classList.length;
+//let squareFour = squareOne[3].classList.length;
+//let squareFive = squareOne[4].classList.length;
+//let squareSix = squareOne[5].classList.length;
+//let squareSeven = squareOne[6].classList.length;
+//let squareEight = squareOne[7].classList.length;
+//let squareNine = squareOne[8].classList.length;
+//console.log(squareOne[0].classList.length);
+//console.log(classList);
+const boxes = document.getElementsByClassName('squares');
+console.log(boxes[0].classList[1]);
+console.log(boxes);
+//Clicks
+const showSquareClicks = (event) => {
+    const classList = event.target.classList;
+    const location = classList[1];
```

Comment:
Note that if other classes got shoved into the classList array later, the one you're looking for might not be at position 1. Consider using an id for this attribute